src/SDL_ttf.c

```diff
@@ -247,7 +247,7 @@ struct TTF_Font {
     int lineskip;
 
     // The font style
-    int style;
+    Uint32 style;
```

Comment:
Shouldn't this be TTF_FontStyleFlags? And the same for prev_style below?

Reply:
I've been thinking about it. Wasn't sure it wouldn't break anything.